.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,32 @@
+on: [push]
+
+name: Code Coverage
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --all-features --no-fail-fast
+        env:
+          CARGO_INCREMENTAL: '0'
+          RUSTFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Zno-landing-pads'
+      - name: Generate coverage report
+        id: coverage
+        uses: actions-rs/grcov@v0.1
+      - name: list all /tmp files
+        run: ls -la /tmp
```

Comment:
I'm wondering about the use of listing all `/tmp` files.

Reply:
Just junk, I'll remove it